src/elements/content-sidebar/AddTaskButton.js

```diff
@@ -44,18 +46,29 @@ class AddTaskButton extends React.Component<Props, State> {
     handleModalClose = () => {
         const { onTaskModalClose } = this.props;
         this.setState({ isTaskFormOpen: false, error: null });
+        if (this.buttonRef) {
+            this.buttonRef.focus();
+        }
         onTaskModalClose();
     };
 
     handleSubmitError = (e: ElementsXhrError) => this.setState({ error: e });
 
+    setAddTaskButtonRef = (ref: { current: null | HTMLButtonElement, focus: Function }) => {
```

Comment:
@alexkrolick thoughts on this bit?

Reply:
shouldn't focus be callable on the HTMLButtonElement? `this.buttonRef.current.focus()`

Reply:
Looks like we are kind of mixing React.createRef with the function pattern here - should use one or the other.

Reply:
@alexkrolick Updated to use React.createRef.